docs/DeveloperGuide.md

```diff
@@ -312,10 +329,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. NUSContacts shows an error message.
 
       Use case resumes at step 2.
 
+**UC3 Find a tutor by name**
+
+**MSS**
+
+1.  User requests to find a tutor
+2.  NUSContacts prompts search criterion (name or course code)
```

Comment:
Can confuse readers as the description of UC3 (at line 336) states to find by name, not course code.

Reply:
Added UC4 `find by course code`  to make UC3 less confusing